packages/admin/admin/src/dataGrid/CrudMoreActionsMenu.tsx

```diff
@@ -52,9 +52,11 @@ interface CrudMoreActionsGroupProps {
 function CrudMoreActionsGroup({ groupTitle, children, menuListProps, typographyProps }: PropsWithChildren<CrudMoreActionsGroupProps>) {
     return (
         <>
-            <Typography variant="overline" color={(theme) => theme.palette.grey[500]} sx={{ padding: "20px 15px 0 15px" }} {...typographyProps}>
-                {groupTitle}
-            </Typography>
+            {groupTitle && (
```

Comment:
Could work here with the Early return pattern, so you have no conditional rendering at the code and make the code better readable.